_security/configuration/tls.md

```diff
@@ -126,16 +126,18 @@ plugins.security.nodes_dn:
 If your node certificates have an Object ID (OID) identifier in the SAN section, you can omit this configuration.
 
 
-## Configuring admin certificates
+## Configuring super admin certificates
```

Comment:
```suggestion
## Configuring admin certificates
```

Reply:
Switching this back for now to fix the build errors. I can rename this section in a separate PR.